matplotlib-tests/bld.bat

```diff
@@ -0,0 +1 @@
+xcopy lib\matplotlib\tests %SP_DIR%\matplotlib /S
```

Comment:
Have you tested this?  I only ask because I ALWAYS get xcopy syntax wrong, and end up copying all files to the given folder, rather than the folder itself.

I find it more intuitive to use robocopy, or move for single folders:

    move lib\matplotlib\tests %SP_DIR%\matplotlib\

Move might be faster, too, but probably not really something to optimize for.